### PR TITLE
tar: Add French translation

### PR DESCRIPTION
**Pull request by jabesq**
Signed-off-by: Hugo D. (jabesq) <jabesq@gmail.com>

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.fr/common/tar.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+- Désarchive une archive (compressée) dans le dossier courant :
+
+`tar xf {{source.tar[.gz|.bz2|.xz]}}`
+
+- Extraie une archive dans un dossier cible :
+
+`tar xf {{source.tar}} -C {{dossier}}`
+
+- Créer une archive compressée, en utilisant le suffix de l'archive pour déterminer le programme de compression :
```
Comment (mebeim) [inline pages.fr/common/tar.md]:
```suggestion
- Créer une archive compressée, en utilisant le suffixe de l'archive pour déterminer le programme de compression :
```

File: pages.fr/common/tar.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# tar
+
+> Utilitaire d'archivage.
+> Souvent combiné avec une méthode de compression, tel que gzip ou bzip.
```
Comment (julienc91) [inline pages.fr/common/tar.md]: "tel que" fait référence à "une méthode de compression", et est donc à mettre au féminin ("telle que")

File: pages.fr/common/tar.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+> Plus d'informations : <https://www.gnu.org/software/tar>.
+
+- Créer une archive à partir de fichiers :
+
+`tar cf {{cible.tar}} {{fichier1}} {{fichier2}} {{fichier3}}`
+
+- Créer une archive gzippé :
```
Comment (julienc91) [inline pages.fr/common/tar.md]: Si on utilise "gzip" comme un participe passé, il faut alors accorder et écrire "gzippée". Ou sinon utiliser "gzip" comme un adjectif invariable, auquel cas ce serait "une archive gzip".

File: pages.fr/common/tar.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# tar
+
+> Utilitaire d'archivage.
+> Souvent combiné avec une méthode de compression, tel que gzip ou bzip.
+> Plus d'informations : <https://www.gnu.org/software/tar>.
+
+- Créer une archive à partir de fichiers :
+
+`tar cf {{cible.tar}} {{fichier1}} {{fichier2}} {{fichier3}}`
```
Comment (antoine-amara) [inline pages.fr/common/tar.md]: please, don't change the command format. the original one is:
`tar cf {{target.tar}} {{file1 file2 file3}}`, so I think it is better to keep `{{file1 file2 file3}}` format into your translation:
```suggestion
`tar cf {{cible.tar}} {{fichier1 fichier2 fichier3}}`
```

File: pages.fr/common/tar.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+- Créer une archive gzippé :
+
+`tar czf {{cible.tar.gz}} {{fichier1}} {{fichier2}} {{fichier3}}`
+
+- Désarchive une archive (compressée) dans le dossier courant :
```
Comment (antoine-amara) [inline pages.fr/common/tar.md]:
```suggestion
- Extraie une archive (compressée) dans le dossier courant :
```

**Pull request closed without merging**